test/unit/validations/profiles.spec.ts

```diff
@@ -1,474 +1,886 @@
-              }
-            }
-          ]
-        },
-        timestamp: ADR_75_TIMESTAMP + 1
-      })
-      const deployment = buildDeployment({ entity })
+  it('should call validateAfterADR75', async () => {
+    await wearableUrnsValidateFn(deployment)
+    expect(mockValidateAfterADR75).toHaveBeenCalledWith(expect.any(Function))
+  })
+
+  describe('and all wearables are valid URNs', () => {
+    it('should return ok', async () => {
+      const result: ValidationResponse = await wearableUrnsValidateFn(deployment)
+      expect(result.ok).toBe(true)
+    })
+  })
 
-      const result = await wearableUrnsValidateFn(deployment)
+  describe('and wearables include old emotes', () => {
+    beforeEach(() => {
+      deployment.entity.metadata.avatars[0].avatar.wearables = [
+        'dance',
+        'wave',
+        'urn:decentraland:matic:collections-v2:0xf6f601efee04e74cecac02c8c5bdc8cc0fc1c721:0'
+      ]
+    })
 
-      expect(result.ok).toBeTruthy()
+    it('should return ok', async () => {
+      const result: ValidationResponse = await wearableUrnsValidateFn(deployment)
+      expect(result.ok).toBe(true)
     })
+  })
 
-    it('When wearable urn is wrong, should return the correct error', async () => {
-      const entity = buildEntity({
-        type: EntityType.PROFILE,
-        metadata: {
-          avatars: [
-            {
-              avatar: {
-                wearables: ['urn:decentraland:tucu-tucu:base-avatars:tall_front_01']
-              }
-            }
-          ]
-        },
-        timestamp: ADR_75_TIMESTAMP + 1
-      })
-      const deployment = buildDeployment({ entity })
+  describe('and a wearable has an invalid URN', () => {
+    let invalidPointer: string
 
-      const result = await wearableUrnsValidateFn(deployment)
+    beforeEach(() => {
+      invalidPointer = 'urn:decentraland:invalid'
+      deployment.entity.metadata.avatars[0].avatar.wearables = [invalidPointer]
+    })
 
-      expect(result.ok).toBeFalsy()
+    it('should return an error with the invalid pointer', async () => {
+      const result: ValidationResponse = await wearableUrnsValidateFn(deployment)
+      expect(result.ok).toBe(false)
       expect(result.errors).toContain(
-        'Each profile wearable pointer should be a urn, for example (urn:decentraland:{protocol}:collections-v2:{contract(0x[a-fA-F0-9]+)}:{name}). Invalid pointer: (urn:decentraland:tucu-tucu:base-avatars:tall_front_01)'
+        `Each profile wearable pointer should be a urn, for example (urn:decentraland:{protocol}:collections-v2:{contract(0x[a-fA-F0-9]+)}:{name}). Invalid pointer: (${invalidPointer})`
       )
     })
   })
 
-  describe('Profile group', () => {
-    it('When entity type is not profile, should not return errors', async () => {
-      const entity = buildEntity({
-        type: EntityType.SCENE,
-        metadata: VALID_PROFILE_METADATA,
-        timestamp
+  describe('and if the deployment is after the ADR 244 timestamp', () => {
```

Comment:
We are not taking into account the timestamps validations wrappers on this test suite because they are mocked, should we still add these cases here?

Reply:
The validation code at the line [102](https://github.com/decentraland/content-validator/blob/924b23f709a5a7aa89d759849f49c659d5b9c630/src/validations/profile.ts#L102-L103) explicitly checks for the timestamp of this ADR, that's why the test is doing an assertion over that case.